README.md

```diff
@@ -253,4 +255,6 @@ See [the list of releases][releases] to find out about feature changes.
 
 [sonobuoyconfig]: https://sonobuoy.io/docs/sonobuoy-config
 
-[strategy]: https://sonobuoy.io/docs/strategy
\ No newline at end of file
+[strategy]: https://sonobuoy.io/docs/strategy
+
+[aggregator-permissions]: https://github.com/vmware-tanzu/sonobuoy/blob/main/site/content/docs/main/aggregator-permissions.md
```

Comment:
This URL shouldn't be the github one; it should be the docs site one.

See the above links for examples. Remember to also just edit the single README and then run the `scripts/sync_readme.sh` script